gammapy/datasets/map.py

```diff
@@ -439,6 +438,17 @@ def __init__(
         else:
             self._meta = meta
 
+    @property
+    def _psf_kernel(self):
+        """Precompute PSFkernel if possible"""
+        if self.psf and self.psf.psf_map.geom.to_image().data_shape == (1, 1):
```

Comment:
Maybe we can introduce a convenience property on the `Map` object here. Such as `Map.has_only_one_spatial_bin` (I'm open to better suggestion for names...:-)

Reply:
I called it `single_spatial_bin` so it's shorter